### PR TITLE
add detail page for editing system user 

### PR DESCRIPTION
**Pull request by mgunnerud**
## Description
- Implement new design to avoid accordions inside accordion
- Change tag component to chip component to enable and disable rights actions 
- Refactor `<ActionBar>` implementation

## Related Issue(s)
- #158 

## Verification
- [x] **Your** code builds clean without any errors or warnings
- [x] Manual testing done (required)
- [x] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [x] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: frontend/src/features/rightsincludedpage/RightsIncludedPageContent.tsx [hunk 1]
```diff
@@ -48,56 +38,35 @@ export const RightsIncludedPageContent = () => {
...
     };
 
-    void dispatch(postNewSystemUser(PostObjekt));
-
-    // Clean up local State variables before returning to main page
-    setIntegrationName('');
-    setDescriptionEntered('');
-    setSelectedSystemType('');
+    void dispatch(postNewSystemUser(PostObjekt)).then(() => {
+      navigate('/' + AuthenticationPath.Auth + '/' + AuthenticationPath.Overview);
```
Comment (Alxandr) [inline frontend/src/features/rightsincludedpage/RightsIncludedPageContent.tsx:51]: Are these parameters validated to be URL safe?

File: frontend/src/features/overviewpage/OverviewPageContent.tsx [hunk 2]
```diff
@@ -65,37 +65,57 @@ export const OverviewPageContent = () => {
...
-            </Heading>
-            {currentRightsCollectionBars}
-          </>
-        </OuterCollectionBar>
+          <div className={cn(classes.accordionContent, { [classes.compact]: isSm })}>
+            <div className={classes.rightsHeader}>
+              <Heading level={3} size='xxsmall' spacing>
+                Systembrukeren har disse rettighetene:
+              </Heading>
+              <Link as={RouterLink} to={`/auth/details/${SystemUser.id}`}>
```
Comment (Alxandr) [inline frontend/src/features/overviewpage/OverviewPageContent.tsx:102]: Are these parameters validated to be URL safe?

File: frontend/src/features/detailpage/DetailPageContent.tsx [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+                  nedenfor for å fjerne de.
+                </Paragraph>
+                <Chip.Group size='small'>
+                  {right.actions.map((action) => {
+                    return (
+                      <Chip.Toggle
+                        key={action.name}
+                        selected={action.on}
+                        onClick={() => {
+                          setSelectedRights((old) => {
```
Comment (Alxandr) [inline frontend/src/features/detailpage/DetailPageContent.tsx:143]: This inner method here is incredibly hard to read. I recommend creating a utility method, so you can do `setSelectedRights(old => toggleRight(old, right, action)`.

**Review by Alxandr — APPROVED** [review]

File: frontend/src/features/detailpage/DetailPageContent.tsx [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+  const [selectedRights, setSelectedRights] =
+    useState<(SystemRegisterProductObjectDTO & { deleted?: boolean })[]>(rightsObjektArray);
+  const [name, setName] = useState<string>(systemUser.integrationTitle);
+
+  return (
+    <div>
+      <Modal ref={deleteModalRef}>
+        <Modal.Header>Sletting av systembruker</Modal.Header>
+        <Modal.Content>
+          {`Er du sikker på at du vil slette systembrukeren "${systemUser.integrationTitle}"?`}
```
[the remark this answers is not in the document]
Comment (mgunnerud) [reply frontend/src/features/detailpage/DetailPageContent.tsx:46]: will be moved to translation file later

**Pull request merged**